aea/cli/common.py

```diff
@@ -498,6 +498,7 @@ def _copy_package_directory(ctx, package_path, item_type, item_name, author_name
         sys.exit(1)
 
     Path(ctx.cwd, "vendor", author_name, item_type_plural, "__init__.py").touch()
+    return Path(dest)
```

Comment:
Function signature return value and docstring has not been updated

Reply:
Dockstring is right there.
https://github.com/fetchai/agents-aea/pull/1131/files/75a03097a0be8b9ecc3a32ddbd2e888f49cad788#diff-7569427e56f7d9e415d5c8f482d1153cR486
Function signature - wasn't there before. Will add.